options/locale/locale_es-ES.ini

```diff
@@ -416,8 +416,8 @@ issues.create_label=Crear etiqueta
 issues.label_templates.title=Carga un conjunto predefinido de etiquetas
 issues.label_templates.helper=Seleccionar un conjunto de etiquetas
 issues.label_templates.fail_to_load_file=Error al cargar el archivo de plantilla de etiqueta '%s': %v
-issues.add_label_at=añadida la etiqueta <div class="ui label" style="color: %s\; background-color: %s">%s</div> %s
-issues.remove_label_at=eliminada la etiqueta <div class="ui label" style="color: %s\; background-color: %s">%s</div> %s
+issues.add_label_at=añadida la etiqueta <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> %s
```

Comment:
These translation files (the ones not en-US) will be overwritten by the CI when merged into master. The changes will have to be added to crowdin

Reply:
RIP.
Okay, I'll back those changes out to clean up the PR and make the changes on crowdin.